src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -6231,6 +6231,36 @@ void Main()
             AssertFormat(expected, code);
         }
 
+        [WorkItem(4240, "https://github.com/dotnet/roslyn/issues/4240")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        public void FormatPredefinedReturnTypesInConversionOperators()
+        {
+            var code = @"class Program
+{
+    public static implicit operator string (Program p) { return null; }
+}";
+            var expected = @"class Program
+{
+    public static implicit operator string(Program p) { return null; }
+}";
+            AssertFormat(expected, code);
+        }
+
+        [WorkItem(4240, "https://github.com/dotnet/roslyn/issues/4240")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        public void FormatOperatorsInConversionOperators()
```

Comment:
If these are supposed to ignore the formatting setting, consider mentioning that in your comment above (SpacingFormattingRule.cs) so it doesn't look like it was omitted by accident.